### ISSUE TITLE
date-input and normalization

### ISSUE THREAD

**Issue by martinantonmueller** [arthur-schnitzler/pmb-service#129]
I write this suggestion once and maybe it's breaking too much code but well – if I could design the input of dates I would keep the current format:

`some written date-text <2021-01-12>`
`1.2.2012`
`2021-12-09`

but if the parser is able to understand and import an iso-date it should empty the written date field. so in all given examples the written-date would be empty except for the first one, where it should be just `some written date-text` without the part in `<…>`

in short: the iso-part from an written date is removed. written date only contains text, when it's different from the iso-date.

currently it is difficult to handle if the written date is the same as the iso-date

**Comment by martinantonmueller** [comment]
After consideration IRL this is not helping. So let's forget it
